<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeedControl</class>
 <widget class="QWidget" name="SpeedControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>522</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
	font-size: 14pt;
    border: 1px solid silver;
    border-radius: 6px;
    margin-top: 14px;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="form_layout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="speed_control_box">
     <property name="title">
      <string>Fan/Pump</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="box_layout">
      <item>
       <widget class="QWidget" name="content_widget" native="true">
        <layout class="QVBoxLayout" name="content_layout">
         <item>
          <layout class="QHBoxLayout" name="control_layout">
           <item>
            <layout class="QVBoxLayout" name="temp_layout">
             <item>
              <widget class="QLabel" name="temp_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">margin-top:14px;</string>
               </property>
               <property name="text">
                <string>TEMP SOURCE</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="temp_combo_box">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">margin-bottom: 14px;</string>
               </property>
               <property name="minimumContentsLength">
                <number>12</number>
               </property>
               <item>
                <property name="text">
                 <string>Liquid</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>CPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GPU</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="toggle_layout"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="profile_layout">
             <item>
              <widget class="QLabel" name="profile_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">margin-top: 14px;</string>
               </property>
               <property name="text">
                <string>SPEED PROFILE</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignHCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="profile_combo_box">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">margin-bottom:14px;</string>
               </property>
               <property name="minimumContentsLength">
                <number>12</number>
               </property>
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Fixed</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Custom</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="graph_frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="graph_layout">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="button_layout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
